src/cbl-mariner/2.0/cross/biarch/amd64-alpine/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-amd64-alpine-local AS crossrootbiarch
+
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-amd64
```

You'll wanna use `local` here, and for the others:

```suggestion
FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-cross-amd64-local
```